src/deltachat.h

```diff
@@ -1031,7 +1063,7 @@ time_t          dc_lot_get_timestamp     (const dc_lot_t*);
  */
 
 #define DC_EVENT_DATA1_IS_STRING(e)  ((e)==DC_EVENT_HTTP_GET || (e)==DC_EVENT_IMEX_FILE_WRITTEN || (e)==DC_EVENT_FILE_COPIED)
-#define DC_EVENT_DATA2_IS_STRING(e)  ((e)==DC_EVENT_INFO || (e)==DC_EVENT_WARNING || (e)==DC_EVENT_ERROR)
+#define DC_EVENT_DATA2_IS_STRING(e)  ((e)==DC_EVENT_INFO || (e) == DC_EVENT_WARNING || (e) == DC_EVENT_ERROR || (e) == DC_EVENT_SMTP_CONNECTED || (e) == DC_EVENT_IMAP_CONNECTED)
```

Comment:
DC_EVENT_SMTP_MESSAGE_SENT is missing. however, maybe we can also use `(e)<1000` if we add all new "INFO" events to the area.